.gitignore

```diff
@@ -1,2 +1,5 @@
 .DS_Store
 .vscode
+img/game_display0.jpg
```

Comment:
For some reason these are in my project. whether I should delete them locally or just add them to the git ignore to prevent accidental upload from a new person is something I'm not up to deciding.

Reply:
I don't have this file but adding it to the .gitignore sounds fine to me.